client/src/components/BaseDatePicker.vue

```diff
@@ -10,29 +10,24 @@
       :disabled="isDisabled"
       input-format="dd.MM.yyyy"
       :upper-limit="upperLimit"
+      :lower-limit="lowerLimit"
     />
+    <!-- :typeable="true" -->
   </div>
 </template>
 
 <script setup>
-// Add the possibility to define a year where the picker should start.
-// No one born in 2021 will enter the comp
 import Datepicker from "vue3-datepicker";
 import { de } from "date-fns/locale";
-import { ref, watch, onMounted } from "vue";
-import { parseISO } from "date-fns";
+import { ref, watch, watchEffect } from "vue";
 const props = defineProps({
   label: {
     type: String,
     required: true,
   },
   modelValue: {
-    type: [String, null],
-    required: true,
-  },
-  initialDate: {
-    type: [String, null],
-    default: null,
+    type: Date,
```

Comment:
is it save to always expect a date? otherwise it will crash.
What will the API send if there is no date value present in db?

Reply:
Yup. See your previous comment. I refactored the sign up process in a way that birthday will be initialzied with null.